glib-macros/src/lib.rs

```diff
@@ -336,17 +336,21 @@ use utils::{parse_nested_meta_items_from_stream, NestedMetaItem};
 /// ```
 #[proc_macro]
 pub fn clone(item: TokenStream) -> TokenStream {
-    // Check if this is an old-style clone macro invocation.
-    // These always start with an '@' punctuation.
-    let Some(first) = item.clone().into_iter().next() else {
-        return syn::Error::new(Span::call_site(), "expected a closure or async block")
-            .to_compile_error()
-            .into();
-    };
+    if cfg!(feature = "unstable-clone-syntax") {
+        // Check if this is an old-style clone macro invocation.
+        // These always start with an '@' punctuation.
+        let Some(first) = item.clone().into_iter().next() else {
+            return syn::Error::new(Span::call_site(), "expected a closure or async block")
+                .to_compile_error()
+                .into();
+        };
 
-    match first {
-        TokenTree::Punct(ref p) if p.to_string() == "@" => clone_old::clone_inner(item),
-        _ => clone::clone_inner(item),
+        match first {
+            TokenTree::Punct(ref p) if p.to_string() == "@" => clone_old::clone_inner(item),
+            _ => clone::clone_inner(item),
+        }
+    } else {
+        clone::clone_inner(item)
```

Comment:
`clone_old::clone_inner(item)`

You're always calling the new one now :) Same for closure/closure_local below